### DISCUSSION TITLE
可不可以添加锁定图片的功能，就是锁定之后鼠标在下面干什么都不会影响到上面的图片

### DISCUSSION THREAD

**Discussion by Ann605** [Snipaste/feedback#1990] [Ideas]
可不可以添加锁定图片的功能，就是锁定之后鼠标在下面干什么都不会影响到上面的图片，运用场景比如画手在下面临摹图片，或者模型师按照概念图在下面雕塑，而浮动在上面的图片不会被下面的操作影响到，完全锁住。

**Comment by Geeyun-JY3** [comment]
鼠标穿透就是你要的功能：https://docs.snipaste.com/zh-cn/getting-started?id=使贴图鼠标穿透

**Comment by Ann605** [comment]
哦哦哦！感谢！On Oct 2, 2021 1:37 AM, Geeyun-JY3 ***@***.***> wrote:

鼠标穿透就是你要的功能
